ui/types/vault/utils/camelize-object-keys.d.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+export default function camelizeKeys(object: object): object;
```

Comment:
Not sure this will actually do what you want it to (might complain that the object that got camelized doesn't have a specific property). Might try:
```
camelizeKeys(object: unknown): {
[key: string]: unknown
}
```
That way it could be cast as a known thing later. But seems non-critical for our usage now

Reply:
Thank you! This didn't seem....right? But I wasn't sure how to approach